src/tests/ftest/soak/harassers.yaml

```diff
@@ -8,8 +8,11 @@ hosts:
 orterun:
   allow_run_as_root: true
 # This timeout must be longer than the test_timeout param (+15minutes)
-# 12 hour test
-timeout: 12H15M
+timeouts:
+  test_soak_online_harassers_24h: 24H30M
```

Comment:
Just a thought. Why can't we have mechanism to calculate timeout based on a testing requirement?
eg: In future, you need to perform a 48hr soak testing, it will be good to have a mechanism within soak framework to automatically set the test timeout to 48hrs or something more based on the requirement during that time...

Reply:
not sure I understand?   The test tag would be test_soak_online_24h and then that would run 24h.  If you want to run soak offline for 12h the test tag for launch would be  test_soak_offline_12h 
I have the same set of tags for soak stress so for instance

to run 48H stress => test_tag for launch is test_soak_stress_48h

to run 24h stress => test tag for launch is test_soak_stress_24h

Reply:
So 24H stress will not be doing same tasks as 48H stress test? 
eg:
   Do test x, y, z ... so on tests for 24H?
   Do the same test x, y, z ... so on tests for 48H?

All I am saying is if we have a single test framework which does x, y, z... so on tests, can we just change the test time alone?

Reply:
NOTE: This is not a requirement now... Something to think about in future... Come with a common framework and add fuzzy logic to it...